trl/models/utils.py

```diff
@@ -132,6 +137,69 @@ def setup_chat_format(
     return model, tokenizer
 
 
+def setup_chat_template(
```

Comment:
Would `clone_chat_template` be a more suitable name?

Reply:
sounds good